class/defaults.yml

```
@@ -1,4 +1,41 @@
 parameters:
   openshift4_splunk_forwarder:
     =_metadata: {}
-    namespace: syn-openshift4-splunk-forwarder
+    multi_instance: true
+    namespace: openshift-logging
+
+    fluentd:
+      loglevel: info
+      replicas: 2
+
+      sharedkey: splunkforwarding
```

This shared key (which probably is a secret) most likely should be stored in Vault too:

```suggestion
      sharedkey: ?{vaultkv:${cluster:tenant}/${cluster:name}/openshift4-splunk-forwarder/${_instance}/fluentd/shared-key
```